src/styles/themes/_default-theme.scss

```diff
@@ -594,8 +609,37 @@
   --button-secondary-disruptive-active-border-color: var(
     --button-secondary-disruptive-active-background-color
   );
+  --button-secondary-disruptive-split-divider-color: var(--disruptive-color-70);
   // ------ Secondary Disruptive Button theme ------
 
+  // ------ System UI Button theme ------
+  --button-system-ui-text-color: var(--grey-color-60);
+  --button-system-ui-background-color: var(--background-color);
+  --button-system-ui-counter-default-text-color: var(--grey-color-80);
+  --button-system-ui-hover-background-color: var(--grey-color-10);
```

Comment:
do you want to add and implement hover text color and set it as `var(--button-system-ui-text-color)` in case people want to customize it?
similar opportunity for other buttons in the PR